docs/openapi/schemas.yaml

```diff
@@ -45,32 +45,31 @@ EmailAddress:
   format: email
   example: 'user@example.com'
 SlackMentionConfig:
+    type: object
+    properties:
+      slack:
+        type: array
+        items:
+          type: string
+HandlerConfig:
   type: object
-  additionalProperties: true
-  properties:
-    slack:
-      type: array
-      items:
-        type: string
-AlertConfig:
-  type: object
-  additionalProperties: true
+  additionalProperties: false
   properties:
-    type:
-      description: The type of the alert
-      type: string
-    byOrg:
-      description: Whether the alert should be sent by organization
-      type: boolean
     mentions:
-      description: The Slack mentions to include in the alert
+      $ref: '#/SlackMentionConfig'
+    excludedURLs:
+      description: URLs to be excluded from the alert
```

Comment:
nitpicking: URLs to be excluded from the handler's processing?

Reply:
Would `manualOverwrites`, `fixedURLs` from broken-backlinks need to be added here as well or would they be allowed under `broken-backlinks` key?

Reply:
I will add them as well

Reply:
@alinarublea would this cover the `manualOverwrites` doc from https://github.com/adobe/spacecat-api-service/pull/387 ? Or can I update the above PR after this one is merged?

Reply:
Yes, it s exactly that, I wont modify it then and I will let you know after I merge